.pipelines/templates/build-dummy-package-for-nuget.yml

```diff
@@ -0,0 +1,33 @@
+# config to pass through parameters and insert the options required to build a dummy nuget package with no operators
+# via build-package-for-nuget.yml
+parameters:
+- name: DoCompliance
+  displayName: Run Compliance Tasks?
+  type: boolean
+  default: true
+
+- name: DoEsrp
+  displayName: Run code sign tasks? Must be true if you are doing an OnnxRuntime extensions release.
+  type: boolean
+  default: true
+
+- name: IsReleaseBuild
+  displayName: Is this a release build? Set it to true if you are doing an OnnxRuntime extensions release.
+  type: boolean
+  default: false
+
+- name: NugetVersionSuffix
+  displayName: Update nuget version suffix (e.g. alpha/beta/rc, only if publishing to nuget.org, otherwise leave as "none").
+  type: string
+  default: none
+
+stages:
+- template: build-package-for-nuget.yml
+  parameters:
+    IsDummyPackage: true
+    DoCompliance: ${{ parameters.DoCompliance }}
+    DoEsrp: ${{ parameters.DoEsrp }}
+    IsReleaseBuild: ${{ parameters.IsReleaseBuild }}
+    NugetVersionSuffix: ${{ parameters.NugetVersionSuffix }}
```

Comment:
perhaps we can add a flag to the version suffix for the dummy package for clarity

Reply:
What would the flag be? The dummy package has a different name so it's differentiated from the 'real' package via that already.